incubator/honeycomb-agent/README.md

```diff
@@ -0,0 +1,144 @@
+# Honeycomb Mixin
+> The [Honeycomb Kubernetes agent](https://github.com/honeycombio/honeycomb-kubernetes-agent/tree/master) is a monitoring tool for pods and their containerized applications (e.g. nginx). It continuously observes, parses, and sends their log files to the Honeycomb API, where the data is rendered in a dashboard for real-time observability.
+
+## Overview
+The Ksonnet files in this directory generate a Kubernetes JSON config, which can then be used to deploy the [Honeycomb](https://honeycomb.io) monitoring agent to your cluster.
+
+The Honeycomb agent runs as a DaemonSet that collects `stdout` logs from every node. It also leverages pod labels to support application-specific log parsing (e.g. nginx, MySQL).
+
+*Ksonnet logic is intended to be highly composable.* The files are split as follows:
+
+* `honeycomb-agent.libsonnet`: This contains all the core functionality for the Honeycomb agent (e.g. DaemonSet, Secrets, RBAC, volume mounting), which can be invoked and combined in other Ksonnet config files. Generally, you will not need to modify this unless you want to create custom mixins.
+
+* `examples/daemonset.jsonnet`: This file contains:
+    * "Meta"-config values (e.g. `secret.key`) that will generate distinct Kubernetes config files if modified. These are detailed  the [configuration section](#configuration).
+
+    * A base Honeycomb DaemonSet that has been combined with a volume-mounting mixin. It attaches three volumes: one volume for the Honeycomb configuration and two volumes to record log output.
+
+* `nginx-app.libsonnet`: This file specifies a sample Nginx app that can be used to test the Honeycomb agent. The pods are tagged with the `app=nginx` label. By default, logs are written to `/var/logs` and `/var/lib/docker/containers` directories.
```

Comment:
I might actually even elide where they're mounted, and just say that we're getting the pod logs from `stdout`.